velox/exec/tests/TableScanTest.cpp

```diff
@@ -622,6 +622,36 @@ TEST_F(TableScanTest, count) {
   EXPECT_EQ(numRead, 10'000);
 }
 
+TEST_F(TableScanTest, batchSize) {
+  // One row is larger than 1KB. Total input is less than 10MB.
```

Comment:
Should we also test smaller rows? It would be nice to test batchSize = 1, 10K and something in between.

Reply:
Revised this unit test to test different row numbers.